---
layout: page
<!-- title: About -->
permalink: /about/
---
## Introduction

Hello!  As indicated in my C.V. (http://victoriastuart.ca/cv.pdf) I hold a Ph.D. in Biology (2000; specialization in molecular genetics), augmented with additional computational expertise.

I possess a unique and well-honed wealth of knowledge and experience, reflecting my intellect and drive (passion for knowledge).  The summary below will provide a reasonably complete, if superficial overview of who I am and what drives my professional pursuits.  I am currently free to pursue challenging opportunities that provide a "good fit" with short-term and long-term visions, on both sides.  I seek high-impact, visionary projects.  I am unencumbered by family/residence/debt, so I am free to relocate on short notice.

I possess a wealth of knowledge and experience in molecular genetics, spanning

 * biochemistry; cell biology; metabolic pathways; cellular signaling pathways
 * molecular genetics and genomics, including cancer biology
 * bioinformatics
 * information retrieval and extraction

and more recently programming experience including, in descending order of expertise/experience:

 * Linux super-user
 * command-line, bash scripting expertise
 * Python; virtual environments
 * NLP: natural language programming
 * Apache Solr: high-performance document indexing/storage/search (backend), +
   user GUI (frontend): Carrot2 clustering engine; D3.js visualizations
 * R ("GNU S"/CRAN)
 * graphical knowledge stores/graphs (KG: Neo4j; Cypher query language)
 * some relational databasing (PostgreSQL)
 * some machine learning (computer vision; vector space models; ...)
 * basic, "hands-on" familiarity with:
   ** webdev (HTML/JS/CSS)
   ** Java (some basic scripting; IntelliJ IDEA ...).

I am a moderate contributor to:

 * StackOverflow: https://stackoverflow.com/users/1904943/victoria-stuart
 *        GitHub: https://github.com/victoriastuart
 *  GitHub Gists: https://gist.github.com/victoriastuart

Regarding machine learning (ML), for a period of ~1.5+ years (2015-2017) I immersed myself in that domain (arXiv; reddit ML subreddit; Y Combinator "Hacker News"; RSS feeds).  I can follow the literature, install and debug the major platforms (Theano; Caffe; Torch7; TensorFlow; etc.  I have posted contributions/solutions associated with various GitHub "Issues" and some StackOverflow.com questions, and I can clone and implement basic models as well as follow the recent literature (that is progressing at a staggering pace).

With personal, self-taught ML projects I implemented:

 * an image captioning system
 * a webcam-based personal identification system: persons identified by name
   with faces identified by bounding boxes with the persons name, probability
 * a web-cam-based classifier: backend: ImageNet top-five categories via a 50-
   layer residual neural network (ResNet-50), + web browser frontend.
 * computer vision (webcam) age-gender classifier
 * other experiments ....

Those were fun and rewarding projects, but my primary motivation in developing those skills and awareness was for supplemental, ML-based approaches to biomedical natural language processing (BioNLP) and bioinformatics; including:

 * classification:
   ** RNN/LSTM (recurrent neural nets/long short-term memory-based models)
   ** VSM (vector space models)
   ** dimensionality reduction
   ** ...
 * knowledge discovery:
   ** topic modeling
   ** graph traversal
   ** ...
 * BioNLP:
   ** biomedical named entity recognition (BioNER)
   ** dependency parsing, applied e.g. to: relation extraction (to populate
      RDBMS and derived knowledge graphs; ...)
   ** semantic parsing, applied e.g. to question-answering; ...
 * fact-checking; quality assurance ("noise" issues)

These tools and approaches support my personal and professional goals, summarized below:

 * information retrieval
 * information extraction
 * knowledge stores
 * quality assurance
 * knowledge discovery
 * biomodels

in support of my long-standing and overarching goal of facilitating a greater understanding of functional genomics: the phenotypic and functional expression of the information contained within our genomes.

I envision, ultimately, the creation of virtual networks (pathways; perhaps cells/tissues/organs), amenable to in silico perturbations and interventions for assessing changes in

 * metabolism
 * cellular signaling
 * cellular growth/death
 * pathogenesis

in response to simulated changes in

 * mutations; genomic alterations
 * epigenetic alterations
 * biochemical entities
 * cellular signalling pathways
 * environmental conditions (stressors)
 * ...

that in turn could guide, for example:

 * personalized/and precision medicine (individualized susceptibilities;
   therapeutic interventions; ...)
 * basic research: augmentation of "wet lab" experiments, via identification/
   ranking of genomic "variants-of-interest" (SNPs); ...
 * synthetic biology.

My stepwise approach in this regard has been to model biochemical and molecular biology/genomics data, first/do date as:

 * backend:
   ** indexed literature (Solr)
   ** BioNLP
   ** extract high-quality data/relations to a RDBMS (PostgreSQL)
   ** "on-the-fly" population of a graphical model (Neo4j) from those data, in
      response to user queries
 * frontend:
   ** GUIs: Carrot2 clustering/visualization engine; D3.js visualizations; ...

Subsequent stages could involve

 * dynamically linking those bioentities to biochemical, molecular genetic, and
   biomolecular databases and other data sources
 * constructing in silico models of metabolic and cellular signaling pathways,
   to aid personalized medicine and basic research ...

While those are ambitious goals, all of these aims are fully tractable (mostly in the near-term).

Following a prolonged period of consulting work for the U.S. DoD (see my C.V.), I have been working full-time (self-supported) since 2015 toward these goals.  I believe those aims are also well-aligned with current research areas in biomedical literature processing, bioinformatics, molecular genetics, cancer genomics, pharma and personalized medicine that are of significant academic and commercial interest.

As such, I am challenging opportunities in this regard.

In closing, I possess superior technical (laboratory), communication (verbal; written), record-keeping and leadership skills.  I am an ambitious, imaginative, team-orientated, and reliable individual.

If you find my expertise relevant to this, or perhaps another, position please do not hesitate to contact me.  I am willing and able to relocate, as needed (Canadian citizenship).

Sincerely,

Dr. Victoria A. Stuart, Ph.D.
Vancouver, B.C.

## Miscellany:

My [curriculum vitae](\.\./cv.pdf) (PDF)

E-mail me [here](mailto:Victoria.A.Stuart@gmail.com)
